xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -109,9 +110,55 @@ void CDVDSubtitlesLibass::Configure()
   ass_set_margins(m_renderer, 0, 0, 0, 0);
   ass_set_use_margins(m_renderer, 0);
 
-  // libass uses fontconfig (system lib) by default in some platforms (e.g. linux/android) or as
-  // a fallback for all platforms. It is not wrapped so translate the path before calling into libass
+  // Libass uses system font provider (like fontconfig) by default in some
+  // platforms (e.g. linux/windows), on some other systems like android the
+  // font provider is currenlty not supported, then an user can add his
+  // additionals fonts only by using the user fonts folder.
   ass_set_fonts_dir(m_library, CSpecialProtocol::TranslatePath(userFontPath).c_str());
+
+  // Load additionals Kodi system fonts to Libass
+  CFileItemList items;
+  XFILE::CDirectory::GetDirectory(systemFontPath, items, "", XFILE::DIR_FLAG_DEFAULTS);
+  for (const auto& item : items)
+  {
+    if (!item->m_bIsFolder && CUtil::IsSupportedFontExtension(item->GetLabel()))
+    {
+      std::vector<uint8_t> buffer;
+      XFILE::CFile file;
+      const std::string filepath = item->GetPath();
+      if (file.LoadFile(filepath, buffer) > 0)
+      {
+        std::string fontFamily = FONT::GetFontFamily(buffer);
+        if (!fontFamily.empty())
+        {
+#if LIBASS_VERSION >= 0x01501000
+          ass_add_font(m_library, fontFamily.c_str(), reinterpret_cast<const char*>(buffer.data()),
+                       buffer.size());
+#else
+          ass_add_font(m_library, const_cast<char*>(fontFamily.c_str()),
+                       reinterpret_cast<char*>(buffer.data()), buffer.size());
+#endif
+          if (StringUtils::CompareNoCase(item->GetLabel(), FONT::FONT_DEFAULT_FILENAME) == 0)
+          {
+            m_defaultFontFamilyName = fontFamily;
+          }
+        }
+      }
+    }
+  }
+  if (m_defaultFontFamilyName.empty())
+  {
+    // This should never happen and in this case the name of the font no longer
+    // matters because in any case will be choosen another fallback font
+    CLog::Log(LOGERROR,
```

Comment:
```suggestion
    CLog::LogF(LOGERROR,
```
and remove the `__FUNCTION__`